llvm/lib/Target/AMDGPU/Utils/AMDGPUBaseInfo.cpp

```diff
@@ -1398,16 +1398,25 @@ getIntegerPairAttribute(const Function &F, StringRef Name,
 SmallVector<unsigned> getIntegerVecAttribute(const Function &F, StringRef Name,
                                              unsigned Size,
                                              unsigned DefaultVal) {
+  std::optional<SmallVector<unsigned>> R =
+      getIntegerVecAttribute(F, Name, Size);
+  return R.has_value() ? *R : SmallVector<unsigned>(Size, DefaultVal);
+}
+
+std::optional<SmallVector<unsigned>>
+getIntegerVecAttribute(const Function &F, StringRef Name, unsigned Size) {
```

Comment:
Title doesn't match content, we already have getIntegerPairAttribute

Reply:
I don't follow. This is for `getIntegerVecAttribute`.

Reply:
Oh, the title of the PR. Yeah, it should be `getIntegerVecAttribute`.